oarepo_ui/theme/assets/semantic-ui/less/oarepo_ui/definitions/preview-modal.less

```diff
@@ -0,0 +1,52 @@
+/*******************************
+            Theme
+*******************************/
+
+/*
+  The type is the type of the component as defined in semantic-ui - element, collection, view, module
+  For simple component, 'element' is the correct type
+*/
+@type: "element";
+
+/*
+  This is the name of the component - though not required, keep it the same as the filename
+*/
+@element: "preview-modal";
+
+/*******************************
+
+    DO NOT FORGET !!!
+
+Edit *theme.config* in your site 
+and add (replace 'sample' with the
+name of the component from above)
+
+@sample : 'default';
+
+*******************************/
+
+@import (multiple) "../../theme.config";
+
+/*
+  This is the styling of your component. The class should start with .ui.<component-name>
+
+  Note: use variables here, not the exact values. Placing the values to another file (sample.variables)
+  will allow users to override these values.
+ */
+.ui.modal#preview-modal {
```

Comment:
Why do we need special css rules just for this, isn't the usual SUI Modal already rendered like this? Isn't there any SUI prop to reach desired ui? This modal should look consistenly, just like any other modals used in our UI